projects/packages/search/src/customberg/class-customberg.php

```diff
@@ -122,16 +127,56 @@ public function load_assets() {
 	 * @param string $plugin_base_path - Base path for plugin files.
 	 */
 	public function load_assets_with_parameters( $path_prefix, $plugin_base_path ) {
-		\Jetpack_Admin_Page::load_wrapper_styles();
+		//
+		// Load styles.
+		wp_enqueue_style(
+			'jp-search-configure',
+			plugins_url( $path_prefix . '/jp-search-configure.css', $plugin_base_path ),
+			array(
+				'wp-components',
+				'wp-block-editor',
+			),
+			JETPACK__VERSION
+		);
+
+		//
+		// Load manifest.
+		$manifest_path       = $plugin_base_path . '/build/customberg/jp-search-configure.asset.php';
+		$script_dependencies = array();
+		try {
+			$asset_manifest      = include $manifest_path;
+			$script_dependencies = $asset_manifest['dependencies'];
+		} catch ( \Exception $e ) { // phpcs:ignore
+			// If the manifest can't be read, use fallback dependencies instead.
+			$script_dependencies = array(
+				'lodash',
+				'react',
+				'react-dom',
+				'wp-block-editor',
+				'wp-components',
+				'wp-core-data',
+				'wp-data',
+				'wp-element',
+				'wp-i18n',
+				'wp-polyfill',
+				'wp-primitives',
+				'wp-url',
+				'wp-viewport',
+			);
+		}
```

Comment:
None of this is necessary. Your problem you reported as #22043 is due to your making a mistake above near line 115, changing it to pass a path to the plugin's directory rather than a file within that directory.

`Assets::register_script()` will already read the `.asset.php` file for you once you get the paths fixed so it can find the files, so there's no need to do that here either.

Reply:
This makes a lot of sense, I hadn't noticed that `$relative_to` needed a file/directory within the plugin path 🤦‍♂️. 

Fixed.